.env-cmdrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I believe using `env-cmd` would require us to build different artifacts (deployable zips) per environment.  Best practice is to have a uniform asset that's promoted through all environments.

So we want some configuration solution that is based on inspecting the hostname during run time.

Reply:
git does not recognize deleted files by default. This was just a test file for something else and should not have been committed. This should now be fixed